packages/components/buttongroup/src/ButtonGroup.styles.ts

```diff
@@ -0,0 +1,71 @@
+import { css } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+import type { ButtonGroupVariants } from './types';
+
+const getGroupContentStyle = (variant: ButtonGroupVariants) => {
+  switch (variant) {
+    case 'separate':
+      return {
+        marginLeft: tokens.spacingS,
+        marginRight: tokens.spacingS,
+        '&:first-child': {
+          marginLeft: 0,
+        },
+        '&:last-child': {
+          marginRight: 0,
+        },
+      };
+  }
```

Comment:
I see that besides the default "variant" we only have another case (separate) which makes this switch works like a simple `if (variant === 'separate')`
Shouldn't we replace it with an if statement then to keep things simple? (and if we increase the number of variants we return with the switch)
Or am I missing something here?

Maybe this is just my personal preference